### DISCUSSION TITLE
Error while trying to provision JWT credentials

### DISCUSSION THREAD

**Discussion by mikyll** [Kong/kubernetes-ingress-controller#5146] [Q&A] [answered]
I've been following the guide to setup the [**ACL plugin**](https://docs.konghq.com/kubernetes-ingress-controller/3.0.x/plugins/acl/) and here's my setup:
- local cluster using **minikube** (and minikube tunnel running in background);
- Kong installed;
- echo service from the [example](https://docs.konghq.com/kubernetes-ingress-controller/3.0.x/get-started/services-and-routes/#deploy-an-echo-service);
- 3 routes (to test different plugins and settings): `/echo`, `/lemon`, `/lime`

Everything is fine and goes as expected, until I reach the [**Provision JWT Credentials**](https://docs.konghq.com/kubernetes-ingress-controller/3.0.x/plugins/acl/#provision-jwt-credentials) step. When I enter the following command:
```
kubectl create secret \
   generic admin-jwt  \
   --from-literal=kongCredType=jwt  \
   --from-literal=key="admin-issuer" \
   --from-literal=algorithm=RS256 \
   --from-literal=rsa_public_key="<public_key_generated_from_jwt.io>"
```

I get this error:
```lang-none
error: failed to create secret admission webhook "validations.kong.konghq.com" denied the request: consumer credential failed validation: missing required field(s): secret
```

How can I fix the error and continue with the tutorial?
Is the documentation wrong/outdated?
Should I include any other option/parameter?
Does this depend on my local setup using minikube?

**Comment by programmer04** [comment, marked as answer]
Thanks for reporting, documentation provides the wrong example. It will be improved ([PR for docs with update](https://github.com/Kong/docs.konghq.com/pull/6502)).

On [the page dedicated JWT plugin](https://docs.konghq.com/hub/kong-inc/jwt/#create-a-jwt-credential) you can read warning

![image](https://github.com/Kong/kubernetes-ingress-controller/assets/9593424/54b912ed-93a2-4547-9eb0-9563717efa1e)

so you have to fill `secret` field too with a dummy value because you're using algorithm `RS256`.

> _If the above answers your question please click `Mark as answer` button to increase visibility for others._

**Reply by mikyll** [reply]
Thank you so, that was super quick :)
